onebusaway-android/src/main/res/values/strings.xml

```diff
@@ -564,11 +564,16 @@
     <string name="preferences_arrival_info_style_options_a">OneBusAway classic</string>
     <string name="preferences_arrival_info_style_options_b">Cards</string>
 
+    <string name="preferences_otp_api_servername_title">Trip Planner API Server</string>
```

Comment:
Could you please change "Trip Planner" to "OpenTripPlanner" in both title and summary?